components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -27,6 +28,7 @@ import mozilla.components.lib.state.State
  * @property undoHistory History of recently closed tabs to support "undo" (Requires UndoMiddleware).
  * @property restoreComplete Whether or not restoring [BrowserState] has completed. This can be used
  * on application startup e.g. as an indicator that tabs have been restored.
+ * @property locale The current locale of the app, defaults to null.
```

Comment:
I think we should indicate what `null` means in context of the local, maybe null could mean that is following the system default (Android OS) and none custom local have been change set.